app/assets/stylesheets/styles/components/guides/_guides.css.scss

```diff
@@ -39,13 +39,63 @@
 .guides {
   position: relative;
 
-  .box {
-    margin: 2rem inherit;
-    padding-top: 1rem;
+  .large-guide {
+    padding-top: 20px;
+
+    .canopy{
+    margin-top: 20px;
+    }
+
+    .guide-controls{
+      margin-right: 0px;
+    }
+
+    .header {
+      border-bottom: rgb(148, 148, 148) 3px solid;
+    }
+
+    .guide-name {
+      float: left;
+      margin-right: 20px;
+    }
+
+    .guide-author {
+      font-size: 1rem;
+      color: #666666;
+      float: left;
+      margin-top: 30px;
+    }
   }
 
-  .stages .box {
-    margin-top: 0;
+  .small-guide {
+    .canopy {
+      height: 400px;
+    }
+
+    .guide-name {
+      color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Rule sets should be ordered as follows: @extends, @includes without @content, properties, @includes with @content, nested rule sets<br>Properties should be ordered bottom, color, left, padding-bottom, padding-top, position, text-align, width